mappings/net/minecraft/structure/BastionTreasureData.mapping

```diff
@@ -0,0 +1,2 @@
+CLASS net/minecraft/class_5182 net/minecraft/structure/BastionTreasureData
```

Comment:
`TreasureBastionData` or even `TreasureBastionRemnantData`, to follow how we named village data (`DesertVillageData`, etc)

Reply:
"Desert village" is a type of village, but "treasure bastion" isn't. All bastions have treasure, and this class stores the data for the treasure of the bastion.